roles-in-tech/product-manager.md

```diff
@@ -77,24 +75,24 @@ Use this guide to know more about role of Product Manager.
  
 ### Common Mistakes / Misconceptions
    
-- Overlook the feedback from customers. 
-- Use too much of tools on a single product which waste both time and efforts.
-- Build a Product according to current situation not future needs.
-
+- Overlook the feedback from customers.
+- Tool are important for a PM. But if a PM uses too many tools it will waste both time and efforts.
+- Build a Product according to current situation, not future needs.
+- Fall in love with the problem, not the solution. The problem is the root of everything. If your product doesn’t solve any problems  it’s useless.
 
 ### Guided Practice
 
-Go through the Links mentioned in the Materials sections. Understand whatever mentioned in those turorials and try to cover it all.
-After watching the video and reading the article you will understand Who are Product Manager and what do they do. 
+Go through the Links mentioned in the Materials section. Understand whatever mentioned in those turorials and try to cover it all.
+After watching the video and reading the article you will understand Who Product Manager are and what do they do. 
 
 ### Independent Practice
 
-Learn to [Identify Market Problem](https://learn.marsdd.com/mars-library/identifying-market-problems/) and to [Prioritize Product Requirement](https://learn.marsdd.com/mars-library/prioritizing-product-requirements/) to Build a Successfull Product.
+Learn to [Identify Market Problems](https://learn.marsdd.com/mars-library/identifying-market-problems/) and to [Prioritize Product Requirement](https://learn.marsdd.com/mars-library/prioritizing-product-requirements/) to Build a Successfull Product.
 
 
 ### Challenge
 
-Can you think what kind of challenges does a Product Manageer have to face? 
+What kind of challenges does a Product Manageer have to face? 
 Check out this [blog](https://productcoalition.com/the-6-biggest-challenges-facing-product-management-teams-in-2018-ef9fd9cfdb9a) 
 to know more.
```

Comment:
Make the content of line 96 and 97 in one line.

Reply:
@Kundan28  I made all such sentence in one line